aio/deploy/recommended/kubernetes-dashboard-head.yaml

```diff
@@ -12,31 +12,6 @@
 # See the License for the specific language governing permissions and
```

Comment:
Looks like this file was rebased to some old version.

Reply:
Yeah, not sure from where. :\ PTAL now.